src/frontend/js/settings/settings.prod.ts

```diff
@@ -61,15 +62,7 @@ export const PER_PAGE = {
   useOrdersEnrollments: DEFAULT_PER_PAGE,
 };
 
-let devSettings;
-try {
-  devSettings = require('settings.dev.ts');
-} catch {
-  devSettings = defaultDevSettings;
-}
-
 export const MOCK_SERVICE_WORKER_ENABLED = false;
+export const DEBUG_UNION_RESOURCES_HOOK = false;
 
-export const DEBUG_UNION_RESOURCES_HOOK = process.env.NODE_ENV !== 'production' && false;
-
-export const { CURRENT_JOANIE_DEV_DEMO_USER } = devSettings;
+export const CURRENT_JOANIE_DEV_DEMO_USER: Nullable<DevDemoUser> = null;
```

Comment:
I don't think that variables only for debug or dev purposes should be declared here.